buzzard.py

```diff
@@ -1106,8 +1116,8 @@ def simplify(points, tolerance, highestQuality):
     parser.add_argument('-v', dest='verbose', default=False,
                         help='Verbose mode (helpful for debugging)', action='store_true')
 
-    parser.add_argument('-o', dest='outMode', default='b', choices=['b', 'ls', 'lib', 'ki'],
-                        help='Output Mode (\'b\'=board script, \'ls\'=library script, \'lib\'=library file, \'ki\'=KiCad footprint)')
+    parser.add_argument('-o', dest='outMode', default='b', choices=['b', 'ls', 'lib', 'ki', 'ki-5.1.x'],
+                        help='Output Mode (\'b\'=board script, \'ls\'=library script, \'lib\'=library file, \'ki\'=KiCad nightly(5.99) footprint, \'ki-5.1.x\'=Kicad 5.1.x footprint)')
```

Comment:
We can probably just call this KiCad v6 and KiCad v5. I know v6 isn't out yet, but the file format in Nightly is what v6 will use.

Reply:
Sure